Networking/Networking/Mapper/StoreOnboardingTaskListMapper.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+/// Mapper: StoreOnboardingTask
+///
+struct StoreOnboardingTaskListMapper: Mapper {
+    func map(response: Data) throws -> [StoreOnboardingTask] {
+        let decoder = JSONDecoder()
+        let taskGroup: [StoreOnboardingTaskGroup]
+
+        do {
+            taskGroup = try decoder
+                .decode(StoreOnboardingTaskEnvelope.self, from: response)
+                .group
+        } catch {
+            taskGroup = try decoder.decode([StoreOnboardingTaskGroup].self, from: response)
```

Comment:
is this to support REST API? 🎉

Reply:
Yes. This is needed as we decided to show the onboarding tasks for self-hosted sites as well.